lib/msal-browser/src/crypto/CryptoOps.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { ICrypto, PkceCodes } from "msal-common";
+import { GuidGenerator } from "./GuidGenerator";
+import { Base64Encode } from "../encode/Base64Encode";
+import { Base64Decode } from "../encode/Base64Decode";
+import { PkceGenerator } from "./PkceGenerator";
+import { BrowserCrypto } from "./BrowserCrypto";
+
+export class CryptoOps implements ICrypto {
+
+    private guidGenerator: GuidGenerator;
+    private b64Encode: Base64Encode;
+    private b64Decode: Base64Decode;
+    private pkceGenerator: PkceGenerator;
+
+    constructor() {
+        const browserCrypto = new BrowserCrypto();
```

Comment:
Might as well also hold on to this as `this.browserCrypto`

Reply:
done